### PR TITLE
Open pointer in new browser tab with [Cmd] click

### LINKED ISSUE #1711: Open pointer in new browser tab with [Cmd] click
## Feature Suggestion

Open a pointer in a new browser tab when clicking on it while holding the [Cmd] button.

The purpose is that while the dashboard already allows to follow a pointer and jump back with the "Back" button, the previous page is newly fetched from the server and scrolled to top. So this type of navigation cannot be practically used to follow multiple pointers.

### PR DESCRIPTION
**Pull request by fn-faisal**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-dashboard/issues/1711).

### Issue Description
Open a pointer in a new browser tab when clicking on it while holding the [Cmd] button.

Closes #1711.

### Approach
Upon clicking on a pointer cell with CMD key held, the relation is opened in a new tab. In addition to this, for windows, an option to open the relation in new tab is added to the ContextMenu.
Screenshot 2021-08-18 at 2 25 24 AM

### TODO:
- [ ] Add changelog entry

### CODE REVIEW

**Comment by mtrezza** [comment]
Thanks for opening this PR so quickly!

- The context menu work fine.
- When I Cmd+click on the pointer, a new tab opens with the pointer object, but the original tab also navigates to the pointer object. Could you look into that?

**Comment by fn-faisal** [comment]
> Thanks for opening this PR so quickly!
> 
> * The context menu works fine.
> * When I Cmd+click on the pointer, a new tab opens with the pointer object, but the original tab also navigates to the pointer object. Could you look into that?

@mtrezza I have updated the PR.

File: Parse-Dashboard/parse-dashboard-config.json [hunk 1]
```diff
@@ -2,13 +2,15 @@
   "apps": [
     {
       "serverURL": "http://localhost:1338/parse",
-      "appId": "hello",
```
Comment (mtrezza) [inline Parse-Dashboard/parse-dashboard-config.json:5]: Can you please revert? I already deleted the index.html file.

**Review by mtrezza — CHANGES_REQUESTED** [review]

File: Parse-Dashboard/parse-dashboard-config.json (hunk 1 above)
Comment (fn-faisal) [reply Parse-Dashboard/parse-dashboard-config.json]: I've reverted these changes.

**Comment by mtrezza** [comment]
This looks good! Thanks for the fixes, could you just add a changelog entry and we should be good to merge this one.

> * Added `Cmd`-click on pointer to open link in new browser tab (fn-faisal) [#1757](https://github.com/parse-community/parse-dashboard/pull/1757)

**Pull request merged**